torch/distributed/_tensor/sharding_prop.py

```diff
@@ -170,8 +172,14 @@ def spec_to_strategy(spec: object) -> object:
                 reshard_schema._inplace_rewrap_schema_suggestion(op_schema)
                 suggestion_schema = [reshard_schema]
 
+            output_spec: OutputSpecType = output_strategy.output_spec
+            if op_schema.op == aten.native_dropout.default:
+                # for ops return multiple tensors, make output spec return same spec
+                # returned from the op strategy
+                output_spec = (output_strategy.output_spec, output_strategy.output_spec)
```

Comment:
Can we have this in the random_op_strategy?

Reply:
Just tried, but it's a bit non-trival to make the `output_spec` to be a tuple of dtensor spec and resharding becomes complicated. I'll try to refactor this in a more canonical way